contracts/exchangeV3/DFSExchangeCore.sol

```diff
@@ -204,7 +201,7 @@ contract DFSExchangeCore is DFSExchangeHelper, DSMath, DFSExchangeData, Exchange
                 feeAmount = _amount / 10;
             }
 
-            address walletAddr = feeRecipient.getFeeAddr();
+            address walletAddr = FeeRecipient(FEE_RECIPIENT_ADDRESS).getFeeAddr();
```

Comment:
unnecessary change that would mean we should redeploy dfs sell?

Reply:
Action inherits 2 contracts with feeRecipient, does not mean we need to redeploy dfs sell as the code is inherited into the action